.github/workflows/firefox-ios-import-strings.yml

```
@@ -60,10 +76,15 @@ jobs:
           echo "branch_name=string-import-$current_branch-$current_date" >> $GITHUB_ENV
           echo "pr_title=Localize [$version] String import $current_date" >> $GITHUB_ENV
           echo "pr_body=This automated PR imports string changes into branch '$current_branch'" >> $GITHUB_ENV
-        fi
+        fi    
 
+    - name: Prepare Xliff files to import
+      run: |
+        # Change all the Xliff files original from en.lproj to en-US.lproj so they can be imported correctly
+        python3 "firefoxios-l10n/.github/scripts/rewrite_original_attribute.py" --path "firefoxios-l10n"
```

```suggestion
        python3 firefoxios-l10n/.github/scripts/rewrite_original_attribute.py --path firefoxios-l10n
```

Don't really need the quotes.